lib/ember-test-helpers/build-registry.js

```diff
@@ -47,6 +47,14 @@ export default function(resolver) {
     registry = new Ember.Registry({
       fallback: fallbackRegistry
     });
+
+    // these properties are set on the fallback registry by `buildRegistry`
+    // and on the primary registry within the ApplicationInstance constructor
+    // but we need to manually recreate them since ApplicationInstance's are not
+    // exposed externally
+    registry.normalizeFullName = fallbackRegistry.normalizeFullName;
+    registry.makeToString = fallbackRegistry.makeToString;
```

Comment:
@dgeb - Yes, I saw that and decided to match `ApplicationInstance`'s `init`.  This is *exactly* why I am suggesting a `wrapAsFallback` method to `Ember.Registry` that is aware of these things.  It seems "wrong" that `Ember.Application` or `ApplicationInstance` (and ember-test-helpers) have to do the internal bookkeeping.